### PR TITLE
Comment DELETE operation

### PR DESCRIPTION
**Pull request by N-Nagorny**
Added a comment saying that DELETE turns off Sink Metadata Processing.

### CODE REVIEW

File: docs/1.2. Behaviour.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ The Sink Metadata Processing API provides a mechanism to change the settings ass
 
 ## Senders
 
-`/senders/{senderId}/media-profiles` allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender and let the underlying component control which it choose exactly which format to use. A client can also pass a single Media Profile to specify the exact format.
+`/senders/{senderId}/media-profiles` allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender, let the underlying component choose exactly which format to use at the moment and dictate that format changes which are not consequent to Media Profiles change MUST lay within Media Profiles set. A client can also pass a single Media Profile to specify the exact format.
```
Comment (prince-chrismc) [inline docs/1.2. Behaviour.md]: This is from #13 ?

Comment (N-Nagorny) [reply docs/1.2. Behaviour.md]: Yes. I thought it's just extra to #13 but it may be redundant.

Comment (N-Nagorny) [inline docs/1.2. Behaviour.md]:
```suggestion
`/senders/{senderId}/media-profiles` allows the client to pass an array of [Media Profiles](1.0.%20Overview.md#media-profile) to a Sender and let the underlying component control which it choose exactly which format to use. A client can also pass a single Media Profile to specify the exact format.
```

Comment (N-Nagorny) [reply docs/1.2. Behaviour.md]: Moved this out.

**Pull request merged**